hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/io/TestHoodieTimelineArchiver.java

```diff
@@ -472,6 +475,12 @@ public void testArchiveCommitSavepointNoHole() throws Exception {
     HoodieTable table = HoodieSparkTable.create(cfg, context);
     HoodieTimelineArchiver archiver = new HoodieTimelineArchiver(cfg, table);
 
+    if (enableMetadataTable) {
+      // Simulate a compaction commit in metadata table timeline
```

Comment:
do we need to repeat the comment block everywhere?

Reply:
It's mainly for the clarity of reading the workflow of the tests.  If it's verbose, I can remove them.